apps/ledger-live-desktop/src/newArch/features/DynamicContent/components/LogContentCardWrapper.tsx

```diff
@@ -20,24 +25,42 @@ const LogContentCardWrapper: React.FC<LogContentCardWrapperProps> = ({
 }) => {
   const ref = useRef<HTMLDivElement>(null);
   const isTrackedUser = useSelector(trackingEnabledSelector);
+  const anonymousUserNotifications = useSelector(anonymousUserNotificationsSelector);
+  const dispatch = useDispatch();
 
   const currentCard = useMemo(() => {
     const cards = braze.getCachedContentCards().cards;
     return cards.find(card => card.id === id);
   }, [id]);
 
   useEffect(() => {
-    if (!currentCard || !isTrackedUser) return;
+    if (!currentCard) return;
 
     const intersectionObserver = new IntersectionObserver(
       ([entry]) => {
         if (entry.intersectionRatio > PERCENTAGE_OF_CARD_VISIBLE) {
-          braze.logContentCardImpressions([currentCard]);
-          track("contentcard_impression", {
-            id: currentCard.id,
-            ...currentCard.extras,
-            ...additionalProps,
-          });
+          console.log("hereee");
```

Comment:
🗑️

Reply:
as the PR work is not done yet  (cleaning up offline notification), i would mark it as draft. (already deleted)